submissions/evgeniy24/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,488 @@
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
+                    </label>
+                  </li>
+                </ul>
+                <ul class="row-number">
+                  <li>1</li>
+                  <li>2</li>
+                  <li>3</li>
+                  <li>4</li>
+                  <li>5</li>
+                  <li>6</li>
+                </ul>
+              </div>
+            </form>
+          </section>
+          <button class="booking__button">BOOK NOW</button>
```

Comment:
Don't use visual representing in markup. Use CSS for it.

Reply:
Done, see changes after a new commit!